content/about/use-case/_index.fr.md

```diff
@@ -5,24 +5,104 @@ description: Applications et axes directeurs
 weight: 60
 ---
 
-Le processus de planification de la circulation d’un train demande **10 ans d’anticipation** et implique la collaboration de différents services et acteurs pour transformer les besoins de mobilité des clients : **autorités organisatrices (AO)** et **entreprises ferroviaires (EF)** en **service annuel (SA)**.
+![OSRD presentation](osrd_presentation.png)
 
-Le projet OSRD à l’ambition de répondre aux besoins de simulation qui couvrent la totalité du processus, de la conception du plan de transport jusqu’à sa mise en exploitation.
+Le projet a pour ambition de répondre à une large gamme de besoins autour de la plannification ferroviaire :
+ - [conception des plans de transport](#conception-des-plans-de-transport)
```

Comment:
&nbsp ; ; at the end of each list item, period "." at the end of the last list item.